PowerShell/README.md

```diff
@@ -41,6 +41,25 @@ New-AWSPowerShellLambda|Used to create an initial PowerShell script that is base
 Publish-AWSPowerShellLambda|Publishes a given PowerShell script to Lambda.
 New-AWSPowerShellLambdaPackage|Creates the Lambda deployment package that can be used in a CI/CD system for deployment.
 
+## Troubleshooting
+### Enabling Debug output
+In PowerShell, [Write-Debug](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/write-debug?view=powershell-7.4) CmdLet could be used to write debug message to the console. However, by default, debug messages are not displayed in the console, but you can display them by using the **Debug** parameter or the **$DebugPreference** variable.
+
+Kindly note that the default value of [DebugPreference](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_preference_variables?view=powershell-7.4#debugpreference) variable is `SilentlyContinue`, which means the debug message isn't displayed and execution continues without interruption. The `-Debug` parameter could be used to override the `$DebugPreference` value.
+
+Enabling output of `Write-Debug` to CloudWatch logs is a 2 step process:
+- In PowerShell Lambda script, 
+  - Either need to set `$DebugPreference = "Continue"` at the beginning of script. Thereafter use `Write-Debug` to output debug messages ; **OR**
+  - Include `-Debug` parameter while executing `Write-Debug` (e.g.` Write-Debug "Testing Lambda PowerShell Write-Debug" -Debug`).
+- At Lambda function level, set the value of `AWS_LAMBDA_HANDLER_LOG_LEVEL` environment variable with value `DEBUG`. This would enable debug logs at Lambda level. This environment variable could be set:
```

Comment:
We should tell users to set the log level in the AWS console or .NET Lambda Tool. We recently shipped new version of the tooling that allows setting the log level. https://github.com/aws/aws-extensions-for-dotnet-cli/blob/master/RELEASE.CHANGELOG.md#release-2024-10-09

Reply:
Actually ignore my previous message. The Lambda API for setting log level is only when the format is set for `JSON`. .NET is special in that it supports log level for text format as well which has existed long before Lambda made log level an API type concern.